src/detection/vulkan/vulkan.c

```diff
@@ -104,8 +104,8 @@ static const char* detectVulkan(FFVulkanResult* result)
     if(instanceVersion.major == 0 && instanceVersion.minor == 0 && instanceVersion.patch == 0)
         instanceVersion.major = 1;
 
-    VkPhysicalDevice physicalDevices[128];
-    uint32_t physicalDeviceCount = (uint32_t) (sizeof(physicalDevices) / sizeof(*physicalDevices));
+    uint32_t physicalDeviceCount = 128;
+    VkPhysicalDevice physicalDevices[physicalDeviceCount];
```

Comment:
Didn't this change make physicalDevices VLA?

Reply:
Yeah you are right, I missed that, will push a fix soon

Reply:
I simply reverted the original commit